src/Components/Subscriptions/Table/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styles from './table.module.css';
+import { FaEdit } from 'react-icons/fa';
+import { FaTimes } from 'react-icons/fa';
+
+const Table = ({ data, deleteItem, setShowModal }) => {
+  const handleDelete = (id) => {
+    deleteItem(id);
+    setShowModal(true);
+  };
+
+  return (
+    <table className={styles.subscriptionsTable}>
+      <thead className={styles.theadContainer}>
+        <tr className={styles.trContainer}>
+          <th>First name</th>
+          <th>Last name</th>
+          <th className={styles.thDaysContainer}>Days</th>
+          <th>Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        {data.map((item) => {
+          return (
+            <tr key={item._id}>
+              <td className={styles.tdFirstNameContainer}>{item.member.firstName}</td>
+              <td>{item.member.lastName}</td>
+              <td>{item.classes.day[0] + ' - ' + item.classes.day[1]}</td>
```

Comment:
Change this line to allow for the case that classes come empty and not break the app, please.
```suggestion
              <td>{item.classes ? item.classes.day[0] + ' - ' + item.classes.day[1] : 'No classes assigned'}</td>
```